miasm2/ir/ssa.py

```diff
@@ -322,3 +322,20 @@ def reassemble_expr(self, e):
                 if id_rhs in self.expressions:
                     todo.add(id_rhs)
         return e
+
+
+class SSAPath(SSABlock):
```

Comment:
I will move it to `analysis`. However, I am not sure if `SSAPath` itself is useless. It is a simple way to provide backward slicing in a few lines for a whole path. Are you sure that I shall remove that?